src/helpers/Bool.cpp

```diff
@@ -0,0 +1,7 @@
+#include "Bool.h"
```

Comment:
These are some helpful constructors for simple ros messages.
Instead of 
`std_msgs::Bool msg; msg.data = true;`
you can use
`std_msgs::Bool msg = Bool(true)`